pages/index.tsx

```diff
@@ -45,8 +55,11 @@ const MainApp = () => {
         <Link href="/api/auth/logout">Logout</Link>
         <br></br>
         <Welcome user={user}/>
-        <Button variant="contained" onClick={displayModal}>Open zone</Button>
-        <InteresZoneView onClose={closeModal} isVisible={openModal} zone={zone!}/>
+        <Button variant="contained" onClick={displayEditModal}>Open zone</Button>
+        <InteresZoneView onClose={closeEditModal} isVisible={openEditModal} zone={zone!}/>
```

Comment:
The `!` character seems redundant, please check.

Reply:
It is mandatory to use ~!~